dashboard-for-desktop/articles/dashboard-designer/designing-dashboard-items/scatter-chart.md

```diff
@@ -29,4 +29,7 @@ This section is divided into the following subsections.
 	Describes how to toggle the chart's orientation.
 * [Labels](scatter-chart/labels.md)
 	
```

Comment:
Please update the following lines as follows:

Line 7: `The topics in this section describe the features available in the **Scatter Chart** dashboard item, and provide information on how to create and customize scatter charts in the **DevExpress Dashboard Suite**.`

`The topics in this section describe the features available in the **Scatter Chart** dashboard item, and explain how to create and customize scatter charts in the **DevExpress Dashboard Suite**.`